Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -1030,14 +1062,166 @@ script:
       name: retries
       required: true
     description: Get PAPI edgehostname creation status command until it is created.
-    name: akamai-get-papi-edgehostname-creation-status-command
-  - name: akamai-acknowledge-warning-command
+    name: akamai-get-papi-edgehostname-creation-status
+  - name: akamai-acknowledge-warning
```

Comment:
This is a correct fix
but that is not possible
because it breaks backward compatibility
therefore, it should be reverted

Reply:
reverted.